packages/embed-examples/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
+  "compilerOptions": {
+    "jsx": "react"
```

Comment:
A path entry to `packages/react-component/src` could be added here to fix typings. See the `/tsconfig.json` at the repo root.

Reply:
Added in https://github.com/genome-spy/genome-spy/pull/255/commits/1abffe2dba59b1c455dbe97aab1276efc23b57f6